ee/vellum_ee/workflows/display/tests/workflow_serialization/generic_nodes/test_adornments_serialization.py

```diff
@@ -237,124 +237,82 @@ class StackedWorkflow(BaseWorkflow):
     exec_config = workflow_display.serialize()
 
     # THEN the workflow display is created successfully
+    inner_stacked_generic_node = [
+        node for node in exec_config["workflow_raw_data"]["nodes"] if node["type"] == "GENERIC"
+    ][0]
     assert not DeepDiff(
         {
-            "workflow_raw_data": {
```

Comment:
I was notified of this issue by noticing this test failing while I was doing: https://github.com/vellum-ai/vellum-python-sdks/pull/1484

This test should focus solely on the node we're serializing, since the rest of workflow raw data is unrelated. I'm going to follow up this PR with doing a similar cleanup refactor for the rest of the tests that were failing for unrelated reasons from the above PR